draft-ietf-wimse-s2s-protocol.md

```diff
@@ -484,6 +484,46 @@ A signed response would be:
 ~~~
 {: title="Signed Response"}
 
+## Comparing the DPoP Inspired Option with Message Signatures
+
+<cref>
+This section is temporarily part of the draft, while we expect the working group
+to select one of these options.
+</cref>
+
+The two options for protecting the workload's traffic vary with respect to implementation
+complexity, extensibility and security. Here is a summary of the main differences between
+{{dpop-esque-auth}} and {{http-sig-auth}}.
+
+- The DPoP-inspired solution is less HTTP-specific, making it easier to adapt for
+other protocols beyond HTTP. This flexibility is particularly valuable for
+asynchronous communication scenarios, such as event-driven systems.
+
+- Message Signatures, on the other hand, benefit from an existing RFC with
+established implementations. This existing groundwork means that this option could
+be simpler to deploy.
```

Comment:
I'd argue that this bullet overstates things a bit. Or more than a bit. This suggestion attempts to tone it down just a little. 
```suggestion
- Message Signatures, on the other hand, benefit from an existing HTTP specific RFC with
some established implementations. This existing groundwork means that this option could
be simpler to deploy, to the extent such implementations are available and easily integrated.  
```

Reply:
Sorry, I let ChatGPT do its thing. Facepalm.